app/grandchallenge/evaluation/models.py

```diff
@@ -1403,6 +1407,35 @@ def metrics_json_file(self):
             if output.interface.slug == "metrics-json-file":
                 return output.value
 
+    @property
+    def missing_metrics_json_file(self):
+        if self.status == self.SUCCESS:
+
+            output_values = [
+                o.value
+                for o in self.outputs.all()
+                if o.interface.slug == "metrics-json-file"
+            ]
+
+            if len(output_values) > 0:
```

Comment:
This will be redundant if you do the filtering in the database that I suggest above.

Reply:
Agree, fixed